wiki/Game_mode/en.md

```diff
@@ -48,6 +50,12 @@ In *osu!catch*, fruits of differing varieties fall from the top of the screen, a
 
 ![Gameplay of osu!mania](/wiki/shared/mania-gameplay.jpg "osu!mania gameplay")
 
+## Custom Game modes
+
+In osu!(Lazer), you can use custom game modes (also called Custom Rulesets) that introduce a new way to play the game using the Beatmap Library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.
+
+The full list of playable Custom Game modes can be found [here](https://github.com/ppy/osu/discussions/13096).
```

Comment:
Using "here" for links is discouraged IIRC, but couldn't see a reference in the Article Styling Criteria.

```suggestion
The full list of playable custom game modes can be found in [this GitHub discussion](https://github.com/ppy/osu/discussions/13096).
```

Reply:
Ahh, didn't know. Will change right now

Reply:
done